sdcm/cluster.py

```diff
@@ -2186,10 +2186,16 @@ def start_scylla(self, verify_up=True, verify_down=False, timeout=300):
     def stop_scylla_server(self, verify_up=False, verify_down=True, timeout=300, ignore_status=False):
         if verify_up:
             self.wait_db_up(timeout=timeout)
-        if not self.is_ubuntu14():
-            self.remoter.sudo('systemctl stop scylla-server.service', timeout=timeout, ignore_status=ignore_status)
-        else:
-            self.remoter.sudo('service scylla-server stop', timeout=timeout, ignore_status=ignore_status)
+        # workaround for issue #7332
```

Comment:
Not only a workaround, but a safer approach, even if the stop "fail" and not stopping gracefully, it starts and we could continue the test.
let's add it to the comment.